_syllabus/w17/week05.md

```diff
@@ -4,8 +4,9 @@ dates: 02/03/2017
 lecturer: mterwil
 homeworkRelease: 2017-02-03 15:00:00
 advancedRelease: 2017-02-03 15:00:00
-solutionRelease: 2017-02-12 15:00:00
+solutionRelease: 2017-02-10 15:00:00
```

Comment:
Yeah, that is probably just a mistake of not re-pulling since started working one week and finished another week.